NerdyNomads/Text-Savvy
# Mm/UI/add workspace list component

## What does this PR do?

- Implements SidebarWrokspaceList component #112, which contains a list of SidebarWorkspaceItem component #110 
- closes #112

### Screenshots (optional)

![Screenshot 2022-03-21 183002](https://user-images.githubusercontent.com/24909769/159379643-eec8f108-a4da-4da6-9685-e9142861e4b2.png)


### Steps to test/review (optional)

- run front end
- See how it renders

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
